keyboards/wanten/readme.md

```diff
@@ -0,0 +1,19 @@
+# wanten
+
+![wanten](https://i.imgur.com/d7o1Edf.png)
+
+an ortho drop in PCB for Van cases
+
+* Keyboard Maintainer: [qpockets](https://github.com/qpockets)
+* Hardware Supported: V1 wanten PCB
+* Hardware Availability: [pocketspace discord server](https://discord.gg/fKbTxraHc6)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```